tests/test_inter_area_agent.py

```diff
@@ -97,9 +96,11 @@ def accept_offer(self, offer_or_id, buyer, *, energy=None, time=None, already_tr
             residual = Offer('res', offer.price, residual_energy, offer.seller,
                              seller_origin='res')
             traded = Offer(offer.id, offer.price, energy, offer.seller, seller_origin='res')
+            self.bids.pop(traded.id, None)
```

Comment:
Why is this needed here ?

Reply:
Its not, remnant of debugging. Will remove it, also in the accept_bid method. Thanks!